src/components/about/about.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { About } from './about';
+
+describe('About Component', () => {
+  it('renders About section with correct content', () => {
+    render(<About />);
+
+    expect(screen.getByText('About')).toBeInTheDocument();
```

Comment:
to test tag name better use:
expect(screen.getByRole('heading')).toHAveTextContent('About');